docs/release-notes/rl-2111.adoc

```diff
@@ -37,6 +37,10 @@ https://github.com/nix-community/home-manager/issues/1906[issue #1906].
 +
 You can replicate your old configuration by moving those options to <<opt-programs.rofi.theme>>. Keep in mind that the syntax is different so you may need to do some changes.
 
+* taskwarrior version 2.6.0 respect XDG Specification for config file now.
+Options <<opt-programs.taskwarrior.config>> and friends now generate config file to
+`${config.xdg.configHome}/task/taskrc` instead of previous `${config.home.homeDirectory}/.taskrc`.
```

Comment:
I think config file location change is a major change. But generated include of `colorTheme` is an implementation detail, thus not worth putting here.